src/components/Search/SearchResultElement.jsx

```diff
@@ -1,49 +1,22 @@
 import React from 'react'
 import { Link } from 'gatsby'
 import PropTypes from 'prop-types'
-import Card from '@material-ui/core/Card'
-import CardContent from '@material-ui/core/CardContent'
-import Typography from '@material-ui/core/Typography'
-import { makeStyles } from '@material-ui/core/styles'
-
-const useStyles = makeStyles({
-  root: {
-    minWidth: 275
-  },
-  bullet: {
-    display: 'inline-block',
-    margin: '0 2px',
-    transform: 'scale(0.8)'
-  },
-  title: {
-    fontSize: 14
-  },
-  pos: {
-    marginBottom: 12
-  }
-})
+import styles from './SearchComponent.module.scss'
 
 const SearchResultElement = ({ element }) => {
-  const classes = useStyles()
   const { slug, title, section, description } = element
   return (
-    <Card container alignItems="center" style={{ width: '100%' }}>
-      <CardContent>
-        <Typography
-          className={classes.title}
-          color="textSecondary"
-          gutterBottom
-        >
-          {section}
-        </Typography>
-        <Typography variant="h6" component="h2">
+    <div className={styles.resultBox}>
+      <div className={styles.secondary}>{section}</div>
+      <div className={styles.title}>
```

Comment:
if it's the title it should be a heading. Please use proper semantic markup for the things you are doing markup for here. Also never use markup for styling, or what was the reason to add in a `b`?